WeatherApp-UIKit/WeatherApp-UIKit/WeatherViewModel.swift

```diff
@@ -87,8 +87,8 @@ struct WeatherViewModel: WeatherViewModelType {
 // MARK: View model without subscriptions
 
 typealias StateMachineViewModelInput = (
-  likeButtonTapped: Observable<Void>,
-  dislikeButtonTapped: Observable<Void>
```

Comment:
This commit should be squshed with the commit where you introduced `StateMachineViewModelInput`